### PR TITLE
Adding basic CloudFoundry support

### PR DESCRIPTION
**Pull request by quickjp2**
I wanted to be able to use CloudFoundry variables, so I wrote a function to be able to parse through a cf environment to gather the needed parameters for the redis service.

I have tested this script and validated it to work in a running cf instance.

### CODE REVIEW

**Comment by quickjp2** [comment]
@technicalpickles would you have some time soon to review this?

Thank you!

**Comment by technicalpickles** [comment]
I'm on paternity leave through the New Year https://github.com/github/hubot/issues/1210

**Comment by quickjp2** [comment]
Totally didn't see that, my bad. Congrats and no rush!

**Comment by quickjp2** [comment]
@technicalpickles Hope you've had a good leave and subsequent months! Wanted to check in and see what the verdict was about this PR, since I have a few more teams at my company wanting to deploy their own hubot in CloudFoundry. 😃 

Thanks!

File: src/redis-brain.coffee [hunk 1]
```diff
@@ -25,6 +30,9 @@ module.exports = (robot) ->
              else if process.env.REDIS_URL?
                redisUrlEnv = "REDIS_URL"
                process.env.REDIS_URL
+             else if process.env.CF_REDIS_SERVICE?
+               redisUrlEnv = "CF_REDIS_SERIVCE"
```
Comment (mose) [inline src/redis-brain.coffee]: There is a typo here on `CF_REDIS_SERIVCE`

**Review by mose — CHANGES_REQUESTED** [review]

**Review by mose — APPROVED** [review]

**Comment by mose** [comment]
hmm, I'm afraid the recent move from coffeescript to pure js included some code changes that make this pull request impossible to merge as-is.

**Comment by quickjp2** [comment]
@mose It's all good, this was submitted way before the move. If you want, you can close this PR, and I'll submit the changes in pure js in a different PR.

**Pull request closed without merging**